index.html

```diff
@@ -5,13 +5,13 @@
 custom_js: mailing-list-signup
 
 intro_heading: We're the GU Tech Society
-intro_leadout: Hacker Olympics is coming up soon! </br> Get your ticket below.
+intro_leadout: CDX is coming soon! </br> Get your ticket below.
```

Comment:
Tickets aren't available, nor are event links, so perhaps best to change it to 'tickets/event available soon' or some such?

Reply:
I prevented it from displaying on the website by commenting the line responsible i.e. <!-- <div class="intro-lead-out">{{page.intro_leadout}}</div> --> but I see it's not the most readable solution. I have now updated the intro_leadout as suggested. Thank you for all the comments!

Reply:
I prevented it from displaying on the website by commenting the line responsible but I see it's not the most readable solution. I have now updated the intro_leadout as suggested. Thank you for all the comments!